test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -631,7 +649,6 @@ def test_transcript_scrubbing
       currency: 'GBP',
       customer: @customer,
       confirmation_method: 'manual',
-      capture_method: 'manual',
```

Comment:
This is a weird thing to delete, I know, but this test was consistently failing because the byte size of the response just _happened_ to be `1737 bytes`. Because the verification value for the test card is `737`, this test kept thinking that the CVV hadn't been correctly filtered out even though it had. I deleted this line just to change up the size of the response.

I'd really love to see a better implementation of these scrubbing tests that isn't so finicky.

Reply:
Might be worth creating an issue for, for posterity?